conda-env-mod

```diff
@@ -58,18 +58,17 @@ function usage()
         Specify only one of the following.
            -n|--name ENV_NAME          : name of the environment.
            -p|--prefix ENV_PATH        : location of the environment.
-
+ 		   -f|--envfile FILE_NAME      : create environment from file
```

Comment:
High-level comment: You need to consider the following scenarios.
1. Is there a filename specified? If so, is that a spec file or a yaml file? YAML files have an environment name, spec files do not.
2. Does the file specify an installation prefix? Did the user provide an installation prefix? Note, command-line prefix overrides what is in the spec file.
3. What is the environment name? Is that in the environment file? Specified by the user? or derived from the prefix?
4. What is the python version? Is that in the environment file? Specified by the user? or derived from the base anaconda version?